packages/realm-app-importer/src/RealmAppImporter.ts

```diff
@@ -131,7 +131,7 @@ export class RealmAppImporter {
     }
 
     private loadStichJson(appTemplatePath: string) {
-        const stitchJsonPath = path.resolve(appTemplatePath, "stitch.json");
+        const stitchJsonPath = path.resolve(appTemplatePath, "config.json");
```

Comment:
I just saw this - why is this needed?

Reply:
The newer versions of the "Stitch" Docker image has changed the naming convention.

Reply:
The changes to RealmAppImporter have been backed out.